src/windows/wslhost/main.cpp

```diff
@@ -74,12 +74,10 @@ class DECLSPEC_UUID("2B9C59C3-98F1-45C8-B87B-12AE3C7927E8") NotificationActivato
         // Log telemetry when a WSL notification is activated, used to determine user engagement for notifications
         WSL_LOG_TELEMETRY("NotificationActivate", PDT_ProductAndServicePerformance, TraceLoggingValue(invokedArgs, "Arguments"));
 
-        // Prepend the executable name to the arguments so getopt can be used to parse the arguments.
-        auto commandLine = wil::GetModuleFileNameW<std::wstring>(wil::GetModuleInstanceHandle());
-        commandLine += L" ";
-        commandLine += invokedArgs;
+        // Prepend the executable name to the arguments.
+        auto commandLine = std::format(L"\"{}\" {}", wil::GetModuleFileNameW<std::wstring>(wil::GetModuleInstanceHandle()), invokedArgs);
```

Comment:
nit: we could avoid having to do this by setting `StartIndex` to 0 when creating the `ArgumentParser`